schedule/yast/maintenance/yast_mru_install_minimal_with_addons_s390x.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: yast_mru_install_minimal_with_addons_s390x
+vars:
+  PATTERNS: base,enhanced_base
+  YUI_REST_API: 1
+schedule:
+  {}
```

Comment:
I guess this file is not needed, with the one above should be ok, right?

Reply:
Removed